zebra-consensus/src/transaction/check.rs

```diff
@@ -21,6 +21,7 @@ pub fn validate_joinsplit_sig(
     joinsplit_data: &JoinSplitData<Groth16Proof>,
     sighash: &[u8],
 ) -> Result<(), TransactionError> {
+    // TODO: batch verify ed25519
```

Comment:
@dconnolly just checking that we don't batch ed25519 already?
Should we open a ticket?

Reply:
We do not, created https://github.com/ZcashFoundation/zebra/issues/1944